app_legacy/Community/Enums/UserGameListType.php

```diff
@@ -0,0 +1,28 @@
+<?php
+
+declare(strict_types=1);
+
+namespace LegacyApp\Community\Enums;
+
+abstract class UserGameListType
+{
+    public const SetRequest = 1;
```

Comment:
For new Enums I'd prefer storing those as strings.
This will also allow for custom lists should we ever want to support that. The values in this enum would just be the default system-provided lists.

Reply:
Not sure I understand this. You want this?
```
public const SetRequest = "SetRequest"
```

Aren't enums supposed to be integers for performance reasons? There's nothing saying we can't assign custom IDs at some fixed offset (100, 256, etc).

Reply:
Performance is not an issue when there's a proper index on the column.

Integers are harder to reason about and not great to maintain, as with most of the existing enums.

I'd go with snake-case (slugged) for the enum values:
```php
public const SetRequest = 'set-request';
public const WantToPlay = 'want-to-play';
public const WantToDev = 'want-to-dev';
```

Reply:
After spending some time with Google, I can see how this will have minimal impact in the database. The individual values won't be on every row. Instead, each row will point to the same value in the index, effectively using the pointer as a numeric enum.

Logic in the code will still have to do string comparisons, but should be much more limited than what would be required to filter the database records.

Reply:
While i'd still use integers for primary keys for consistency's sake, there is nothing preventing to use strings (like uuids, ulids) as primary key to be indexed.
It used to be problematic in the past with some db engines, these days it's just about index size iirc. MySQL got a lot better since then.